src/api-routes/locations.js

```diff
@@ -1,6 +1,6 @@
 import { errorHandler } from 'errors/errors';
-import { getLocations } from '../db/awsES/locations-dao';
-import { serializeLocations } from '../serializers/locations-serializer';
+import { getLocations } from 'db/awsEs/locations-dao';
```

Comment:
The import order should be fixed here: https://github.com/osu-mist/style-guides/blob/master/JavaScript.md#import-order